Apsitvarkom.DataAccess/GoogleGeocoder.cs

```diff
@@ -22,13 +24,17 @@ public GoogleGeocoder(HttpClient httpClient, IApiKeyProvider apiKeyProvider)
         _httpClient = httpClient;
     }
 
-    public async Task<string?> ReverseGeocodeAsync(Coordinates coordinates)
+    public async Task<List<LocationTitle>> ReverseGeocodeAsync(Coordinates coordinates)
```

Comment:
this method is kind of a Frankenstein now and doesn't really comply with single responsibility principle. will maybe look into fixing it.